### ISSUE TITLE
Some issue with running on GPU.

### ISSUE THREAD

**Issue by Voldy1998** [AntonMu/TrainYourOwnYOLO#80]
Before filing a report consider the following two questions:

### Have you checked the [troubleshooting](https://github.com/AntonMu/TrainYourOwnYOLO#troubleshooting) section?Yes

Once you are familiar with the code, you're welcome to modify it. Please only continue to file a bug report if you encounter an issue with the provided code and after having followed the instructions.

If you have followed the instructions exactly, couldn't solve your problem with the provided troubleshooting tips and would still like to file a bug or make a feature requests please follow the steps below.

1. It must be a bug, a feature request, or a significant problem with the documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

------------------------

### System information
- **What is the top-level directory of the model you are using**: 2_Training
- **Have I written custom code (as opposed to using a stock example script provided in the repo)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: windows 10
- **TensorFlow version (use command below)**:  v1.15.0-rc3-22-g590d6eef7e 1.15.0
- **CUDA/cuDNN version**: Cuda 10 and the latest cudnn for cuda 10
- **GPU model and memory**: Nvedia Geforce 940MX and 4GB graphics card
- **Exact command to reproduce**: python Train_YOLO.py

You can obtain the TensorFlow version with

`python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"`

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug or a feature request.
I installed Cuda 10 and corresponding latest cudnn version for cuda 10. However I am getting some weird messages 
![anton](https://user-images.githubusercontent.com/40512980/74060065-33aa8700-4a0f-11ea-8639-c7702a267bfd.PNG)
![anton 2](https://user-images.githubusercontent.com/40512980/74060102-40c77600-4a0f-11ea-9d58-097f18b640a2.PNG)
![anton 3](https://user-images.githubusercontent.com/40512980/74060111-458c2a00-4a0f-11ea-9ec1-f9bbf19a423b.PNG)
![anton 4](https://user-images.githubusercontent.com/40512980/74060123-4a50de00-4a0f-11ea-9853-c6a10d371a46.PNG)
![anton 5](https://user-images.githubusercontent.com/40512980/74060128-4e7cfb80-4a0f-11ea-8cbd-87cba7d58dbb.PNG)
At the 1/51 epoch after 10/11th iteration it displays all the above messages which i am not able to understand. What exactly is the problem here?
Also i think the weights are not getting updated. When i tried testing some images it took the weights which i had got before (when i tried to train it with cpu)
### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.
Attached as images above

**Comment by AntonMu** [comment]
Hi it looks like you don't have enough memory on your GPU available. You could try to [reduce your batch size](https://github.com/AntonMu/TrainYourOwnYOLO/blob/master/2_Training/Train_YOLO.py#L200) to 16 or 8 and see if you still get the same results.

I also encourage you to follow the instructions and avoid spaces in folder names and create a virtual environment. 

Hope this helps!

**Comment by Voldy1998** [comment]
Hi, thanks for the suggestion @AntonMu I did try by changing the batch size to 8. It worked fine till 51st epoch. After that the same issue came up.
![pbm](https://user-images.githubusercontent.com/40512980/74416633-6d461c80-4e6b-11ea-9351-c925d0909f3e.PNG)
Any suggestions?
what's the required memory for graphics card?

**Comment by AntonMu** [comment]
Great to hear! Usually 10GB or more is a standard assumption for running CUDA jobs. And yes, the second step unfreezes more layers and thus needs more memory. You should try 4 then.

**Comment by robisen1** [comment]
if i chane line 240 to if FALSE does this mean that all layers will be unfrozen? my imagery is from a drone and the loss from training is very high. like 80.
